arangod/Graph/Options/TwoSidedEnumeratorOptions.h

```diff
@@ -37,12 +36,14 @@ struct TwoSidedEnumeratorOptions {
 
   ~TwoSidedEnumeratorOptions();
 
-  size_t getMinDepth() const;
-  size_t getMaxDepth() const;
+  [[nodiscard]] size_t getMinDepth() const;
+  [[nodiscard]] size_t getMaxDepth() const;
+  [[nodiscard]] bool isDisjoint() const;
```

Comment:
I do not think this belongs here.

Reply:
Refactored the API. isDisjoint and isSatelliteLeader now part of PathValidatorOptions